<ui version="4.0" >
 <class>GmailNotifierAppletConfig</class>
 <widget class="QWidget" name="GmailNotifierAppletConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Accounts</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" >
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="1" >
         <widget class="KPushButton" name="btnAddModify" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>Add this to the account list</string>
          </property>
          <property name="text" >
           <string>Add</string>
          </property>
          <property name="autoDefault" >
           <bool>true</bool>
          </property>
          <property name="default" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item rowspan="3" row="4" column="0" >
         <widget class="KListWidget" name="listAccounts" >
          <property name="toolTip" >
           <string>Account list</string>
          </property>
          <property name="showDropIndicator" stdset="0" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <layout class="QGridLayout" name="gridLayout_2" >
          <item rowspan="2" row="0" column="0" >
           <widget class="QLabel" name="lblLogin" >
            <property name="text" >
             <string>Login:</string>
            </property>
            <property name="buddy" >
             <cstring>leLogin</cstring>
            </property>
           </widget>
          </item>
          <item rowspan="2" row="0" column="1" colspan="2" >
           <widget class="KLineEdit" name="leLogin" >
            <property name="toolTip" >
             <string>Gmail login name</string>
            </property>
            <property name="whatsThis" >
             <string>whatsThis</string>
            </property>
            <property name="showClearButton" stdset="0" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="lblPassword" >
            <property name="text" >
             <string>Password:</string>
            </property>
            <property name="buddy" >
             <cstring>lePassword</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="3" >
           <widget class="KLineEdit" name="lePassword" >
            <property name="toolTip" >
             <string>Gmail password</string>
            </property>
            <property name="showClearButton" stdset="0" >
             <bool>true</bool>
            </property>
            <property name="passwordMode" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0" >
           <widget class="QLabel" name="lblDisplay" >
            <property name="text" >
             <string>Display (optional):</string>
            </property>
            <property name="buddy" >
             <cstring>leDisplay</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="3" >
           <widget class="KLineEdit" name="leDisplay" >
            <property name="toolTip" >
             <string>Use this if you want to display something other than your login name in the applet</string>
            </property>
            <property name="showClearButton" stdset="0" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item rowspan="2" row="0" column="3" >
           <widget class="QLabel" name="lblGmaildotcom" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>@gmail.com</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="lblLabel" >
            <property name="text" >
             <string>Label (optional):</string>
            </property>
            <property name="buddy" >
             <cstring>leLabel</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="3" >
           <widget class="KLineEdit" name="leLabel" >
            <property name="toolTip" >
             <string>Gmail label (leave empty for inbox)</string>
            </property>
            <property name="showClearButton" stdset="0" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="1" >
         <widget class="KPushButton" name="btnDown" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="toolTip" >
           <string>Move the selected icon down</string>
          </property>
          <property name="text" >
           <string>Down</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" >
         <widget class="KPushButton" name="btnUp" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="toolTip" >
           <string>Move the selected icon up</string>
          </property>
          <property name="text" >
           <string>Up</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="KPushButton" name="btnDelete" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="toolTip" >
           <string>Delete the selected account</string>
          </property>
          <property name="text" >
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <spacer name="verticalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" >
    <widget class="QCheckBox" name="cbDisplayLogo" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="toolTip" >
      <string>Show/hide the Gmail logo in the applet</string>
     </property>
     <property name="text" >
      <string>Display Gmail logo</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <layout class="QFormLayout" name="formLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="lblPollingInterval" >
       <property name="text" >
        <string>Polling interval:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="KIntSpinBox" name="spinPollingInterval" >
       <property name="suffix" >
        <string> mins</string>
       </property>
       <property name="minimum" >
        <number>5</number>
       </property>
       <property name="maximum" >
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="lblBackground" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Background:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="KComboBox" name="comboBackground" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Applet background</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leLogin</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>leLabel</tabstop>
  <tabstop>leDisplay</tabstop>
  <tabstop>btnAddModify</tabstop>
  <tabstop>listAccounts</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnUp</tabstop>
  <tabstop>btnDown</tabstop>
  <tabstop>cbDisplayLogo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
